nrf_security/doc/backends.rst

```diff
@@ -11,16 +11,15 @@ The nrf_security module supports multiple enabled backends at the same time.
 This mechanism is intended to extend the available feature set of hardware-accelerated cryptography or to provide alternative implementations of the Mbed TLS APIs.
 Enabling one or more backends adds more configuration options grouped into classes of cryptographic algorithms.
 
-Note that some cryptographic features are provided as an option regardless of the backend enabled.
-In such cases, the feature is compiled using open-source software from the original Arm Mbed TLS project.
-An example of this is `SHA-512`, which is only accessible using open-source software from the Arm Mbed TLS project.
+Note that some cryptographic features are provided by Mbed TLS even if it is disabled.
+An example of this is `SHA-512`, which is only available in Mbed TLS.
```

Comment:
Statement untrue with new version of nrf_oberon library. This line can now be removed